src/LibraryViewExtensionWebView2/Packages/LibrarieJS/package.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
Why do we need to include a copy of package.json here?

Reply:
Just removed this, you're right, there's no need to include package.json as it is part of the npm package